introduction.md

```diff
@@ -0,0 +1 @@
+Hello! I'm Tunghim Lam. I love software, open source, GitHub and last but not least, Code School!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.